docs/how_to_guides/how_to_use_debugging_solver_wrapper.rst

```diff
@@ -0,0 +1,141 @@
+.. _how_to_use_debugging solver wrapper:
+
+How to use the debugging solver wrapper
+=======================================
+
+Overview
+--------
+
+Debugging a failed initialization or solve can be cumbersome. 
+Further, it is often useful to have the model state (i.e., initial variable values) from before the solve failed.
+The debugging solver wrapper facilitates the following:
+
+(1) Stores initialization information before a failed solve
+
+(2) Upon a failed attempt to solve, the user is routed to an Interactive Python notebook where the restored model state can be accessed, `IDAES' DiagnosticToolbox <https://idaes-pse.readthedocs.io/en/stable/reference_guides/core/util/diagnostics/diagnostics_toolbox.html>` is instantiated to probe the model, and the user can freely apply any other diagnostic utility functions to troubleshoot the problematic model. 
```

Comment:
```suggestion
(2) Upon a failed attempt to solve, the user is routed to an Interactive Python notebook where the restored model state can be accessed, `IDAES' DiagnosticToolbox <https://idaes-pse.readthedocs.io/en/stable/reference_guides/core/util/diagnostics/diagnostics_toolbox.html>`_ is instantiated to probe the model, and the user can freely apply any other diagnostic utility functions to troubleshoot the problematic model. 
```
Link is not working correctly - I think you're just missing this one character

Reply:
Thanks!